examples-of-custom-resources/egress-mtls/README.md

```diff
@@ -0,0 +1,60 @@
+# Egress MTLS
+
+In this example, we deploy a web application, configure load balancing for it via a VirtualServer, and apply an Egress MTLS policy.
+
+## Prerequisites
+
+1. Follow the [installation](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/) instructions to deploy the Ingress Controller.
+1. Save the public IP address of the Ingress Controller into a shell variable:
+    ```
+    $ IC_IP=XXX.YYY.ZZZ.III
+    ```
+1. Save the HTTP port of the Ingress Controller into a shell variable:
+    ```
+    $ IC_HTTP_PORT=<port number>
+    ```
+
+## Step 1 - Deploy a Secure Web Application
+
+Create the application deployment, service and secret:
+```
+$ kubectl apply -f secure-webapp.yaml
+```
+
+## Step 2 - Deploy the Egress MLTS Secret
+
+Create a secret with the name `egress-mtls-secret` that will be used for authentication to the Secure Web Application:
```

Comment:
I think we can use the application here and going forward, similarly to the styling in the other examples:
```suggestion
Create a secret with the name `egress-mtls-secret` that will be used for authentication to application:
```